src/core/polymer.cpp

```diff
@@ -126,9 +127,9 @@ draw_polymer_positions(PartCfg &partCfg, int const n_polymers,
   std::vector<std::vector<Utils::Vector3d>> positions(
       n_polymers, std::vector<Utils::Vector3d>(beads_per_chain));
 
-  auto rng = [mt = std::mt19937{static_cast<unsigned>(seed)},
-              dist = std::uniform_real_distribution<double>(
-                  0.0, 1.0)]() mutable { return dist(mt); };
+  auto mt = Random::mt19937(static_cast<unsigned>(seed));
```

Comment:
Is there a reason why you widened the scope of `mt`?

Reply:
Nice catch, I originally moved `mt` out to discard the first 1 million values, but then I made a wrapper in `random.hpp` to avoid the situation where MT would used without warmup. I'll put it back in the capturing group.